swagger_to_sdk_config.json

```diff
@@ -16,7 +16,7 @@
   },
```

Comment:
@tadelesh @ArcturusZhang as a potential further clean-up we can move this config file under eng directory as well we will just need to update the path at https://github.com/Azure/azure-rest-api-specs/blob/main/specificationRepositoryConfiguration.json#L5.

Reply:
@benbp @weshaggard There is two versions of swagger_to_sdk_config.json. One under repo root (work with initScript.sh) is for track1 and the other under eng folder (work with eng/script/automation_init.sh) is for track2. We can aggregate and rename them, in the meantime, change the sdk automation config in swagger repo.

Reply:
I had opted not to move this json file because most other repositories have one at the root, so I didn't see it was breaking convention like the other files.

Reply:
I'm fine with punting on thus for now but I've definitely been trying to get the other languages to move those config files under the eng folder but I agree it isn't as critical currently and requires other changes outside of this repo to actually move.